rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/BatchJobInfrastructure.java

```diff
@@ -496,7 +497,7 @@ public void configure(Object... parameters) {
                 persistedInfraVariables.put(CREDENTIALS_KEY,
                                             Credentials.getCredentialsBase64((byte[]) parameters[index++]));
             } catch (KeyException e) {
-                throw new IllegalArgumentException("Could not retrieve base64 credentials", e);
+                logger.debug("Could not retrieve base64 credentials", e);
```

Comment:
if credentials are provided and incorrect, maybe we could trigger the error ?

Reply:
you are right